configs/devices/uart_tb.json

```diff
@@ -11,9 +11,10 @@
     "type": "dpi",
     "module": "uart.so",
     "verbose": false,
-    "baudrate": 115200,
+    "baudrate": 9600,
```

Comment:
Can we keep the default 115200 value, otherwise it won't be aligned anymore with the rest (RTL platforms, pulpos and so on) ?

Reply:
ok, done